docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -109,136 +105,17 @@ The following example shows how to quickly remove old data via dropping the part
 DROP TABLE employees_2019_02;
 ```
 
-The following example shows how remove the partition from the partitioned table while retaining access to it as a separate table which enables you to performed operatioins on the data:
+The following example shows how to remove the partition from the partitioned table while retaining access to it as a separate table which enables you to performed operations on the data:
 
 ```sql
 ALTER TABLE employees DETACH PARTITION employees_2019_02;
 ```
 
-The following example shows how to add a new partition to deal with new data by creating an empty partition in the partitioned table:
```

Comment:
Oh you can add back this section. Please only remove the TABLESPACE clause in the CREATE statements.

Reply:
Added